### DISCUSSION TITLE
What happens when already having an instance of ViGEmBus installed?

### DISCUSSION THREAD

**Discussion by cloudropis** [nirgoren/Eddienput#22] [Technical support]
I already have ViGEmBus installed through DS4Windows. Will installing it again through Eddienput break the other software? Will they coexist? Will the installation of ViGEmBus be skipped as it's already installed?

**Comment by nirgoren** [comment]
From our little testing it looks like the installation of ViGEmBus can be skipped in this case.

**Reply by cloudropis** [reply]
Is there an opt-out check in the launcher or something like that? Unfortunately I won't be able to access my PC for a while to see how it looks like
